Oh, Cargo mixes dev-deps...

https://github.com/smol-rs/blocking/actions/runs/8130067781/job/22217900369?pr=57

```
error[E0405]: cannot find trait `AsyncRead` in this scope
   --> src/lib.rs:769:32
    |
769 | impl<T: Read + Send + 'static> AsyncRead for Unblock<T> {
    |                                ^^^^^^^^^ not found in this scope
    |
help: consider importing this trait
    |
87  + use futures_io::AsyncRead;
    |

error[E0[40](https://github.com/smol-rs/blocking/actions/runs/8130067781/job/22217900369?pr=57#step:6:41)5]: cannot find trait `AsyncWrite` in this scope
   --> src/lib.rs:837:33
    |
837 | impl<T: Write + Send + 'static> AsyncWrite for Unblock<T> {
    |                                 ^^^^^^^^^^ not found in this scope
    |
help: consider importing this trait
    |
87  + use futures_io::AsyncWrite;
    |

error[E0405]: cannot find trait `AsyncSeek` in this scope
   --> src/lib.rs:920:32
    |
920 | impl<T: Seek + Send + 'static> AsyncSeek for Unblock<T> {
    |                                ^^^^^^^^^ not found in this scope
    |
help: consider importing this trait
    |
87  + use futures_io::AsyncSeek;
    |
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -103,7 +103,11 @@ use std::env;
 use async_channel::{bounded, Receiver};
 use async_task::Runnable;
 use futures_io::{AsyncRead, AsyncSeek, AsyncWrite};
-use futures_lite::{future, prelude::*, ready};
+use futures_lite::{
+    future::{self, Future},
+    ready,
+    stream::Stream,
+};
 use piper::{pipe, Reader, Writer};
 
 #[doc(no_inline)]

